riskified-sdk/src/main/java/com/riskified/RiskifiedClient.java

```diff
@@ -137,12 +225,36 @@ public Response createOrder(Order order) throws ClientProtocolException, IOExcep
      * @throws ClientProtocolException in case of a problem or the connection was aborted
      * @throws IOException in case of an http protocol error
      * @throws HttpResponseException The server respond status wasn't 200
+     * @throws FieldBadFormatException 
      */
-    public Response submitOrder(Order order) throws ClientProtocolException, IOException, HttpResponseException {
+    public Response submitOrder(Order order) throws ClientProtocolException, IOException, HttpResponseException, FieldBadFormatException {
```

Comment:
This method by default validates ignoreMissing, it has an overload with boolean shouldValidateLikeCreation. if the customer is directly submitting (without create) he should specify second parameter true so the order will be fully validated.